kubernetes_deploy/production/ingress.yml

```diff
@@ -0,0 +1,26 @@
+apiVersion: extensions/v1beta1
+kind: Ingress
+metadata:
+  name: laa-legal-adviser-api
+  namespace: laa-legal-adviser-api-production
+spec:
+  tls:
+  - hosts:
+    - laa-legal-adviser-api-production.apps.live-1.cloud-platform.service.justice.gov.uk
+  - hosts:
+      - laa-legal-adviser-api-production.cloud-platform.service.justice.gov.uk
```

Comment:
_not a blocker_
Should we confirm with Cloud Platform that this is a domain we are allowed to use?

Reply:
What do you mean allowed here, do you mean whether we can publish the domain?

Reply:
Yes, can we use the shorter domain?

Reply:
[Reply](https://mojdt.slack.com/archives/CH6D099DF/p1560167844000900?thread_ts=1554290674.012100&cid=CH6D099DF) from Cloud Platform:

> yes, they're available and permanent

👍 🎉

Reply:
Great, I have asked and Raz from cloud Platforms has told me that they're available and permanent